docs/source/learn/core_notebooks/dimensionality.ipynb

```diff
@@ -9,28 +9,29 @@
    },
```

Comment:
<p>Does this actually render in an online notebook? I only see <code>&lt;graphviz.dot.Digraph at some_address&gt;</code> </p><p>Maybe we need use <code>view</code> and get a base64 binary representation of the png? If this renders fine once compiled into the docs, forget what I said</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc/pull/5746/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='58'/>

Reply:
<p>Yeah it renders, there's a link to the rendered docs in the top message of the PR</p>